server/index.ts

```diff
@@ -1,17 +1,37 @@
-import { createServer } from "http";
+import { graphiqlExpress, graphqlExpress } from "apollo-server-express";
+import * as bodyParser from "body-parser";
+import * as express from "express";
 import * as next from "next";
-import { parse } from "url";
+import * as schema from "./schema";
 
 const port = parseInt(process.env.PORT, 10) || 3000;
 const dev = process.env.NODE_ENV !== "production";
 const app = next({ dev });
 const handle = app.getRequestHandler();
 
 app.prepare().then(() => {
```

Comment:
Could rewrite this to also use async await I guess :)

Reply:
You mean like: 
```typescript
await app.prepare(); 
createServer(...);
...
```

Should ```await``` not be inside an `async` function?

Reply:
Ah yes, woops. Nevermind :)